doc/source/install/build.rst

```diff
@@ -0,0 +1,106 @@
+Build and install
+=================
+
+Once you :doc:`have the source code <get-the-source-code>` and have
+:doc:`activated the virtual environment <virtual-environment>`, you're ready to
+build and install `pycompwa`.
+
+When you install `pycompwa`, you are telling the system where to find it. There
+are two ways of doing this:
+
+(1) by copying the source code and binaries to a folder known to the system
+    (you do this :ref:`with setuptools <setuptools>`)
+(2) by telling the system to directly monitor the :doc:`local repository
+    <get-the-source-code>` as the installation path (we call this
+    :ref:`'developer mode' <install/build:Developer Mode>`).
+
+The second option is more dynamic, because any changes to the source code are
+immediately available at runtime, which allows you to tweak the code and try
+things out. When using the first option, you would have to run ``setuptools``
+again to make the changes known to the system.
+
+
+.. _setuptools:
+
+Using ``setuptools``
+~~~~~~~~~~~~~~~~~~~~
+
+This is easy-peasy! Just navigate to the :doc:`local repository
+<get-the-source-code>` and run:
+
+.. code-block:: shell
+
+  python setup.py install -- -- -j2
+
+where you may change 2 to to the number of cores on your system. The build
+output is written to a folder :file:`_skbuild` and copied to a system folder
+from there.
+
+
+Developer Mode
+~~~~~~~~~~~~~~
+
+In this set-up, we first tell the virtual environment to monitor the source
+code directory as an install directory. So, navigate to the :doc:`local
+repository <get-the-source-code>` then, depending on which :doc:`virtual
+environment </install/virtual-environment>` you chose, do the following:
```

Comment:
You should mentioned that you should be in the base directory of pycompwa

Reply:
https://github.com/ComPWA/pycompwa/pull/83/commits/f2e5fc888c1224afd20f23c2e5e58a8e24426483